src/main/app/src/components/hakutulos/Hakutulos.js

```diff
@@ -20,9 +22,11 @@ import { useStores } from '../../hooks';
 const Hakutulos = observer((props) => {
   const { t, classes } = props;
   const { hakuStore } = useStores();
+  const { filter } = hakuStore;
+  const { koulutustyyppi, koulutusala, opetusKielet, sijainti } = filter;
 
   useEffect(() => {
-    window.scroll(0, 0);
+    window.scroll(0, 170);
```

Comment:
?

Reply:
Focus menee muuten alaspäin, kun lisätään filtteriarvon